src/Log/LogService.php

```diff
@@ -268,8 +270,14 @@ public function getLogConfig($name)
     {
         $config = $this->getConfig();
 
+        if (! isset($config->logging->log->$name) || ! isset($config->logging->log->$name->format)) {
+            $format = $this->getDefaultFormat();
+        } else {
+            $format = $this->addRunIdToFormat($config->logging->log->$name->format);
+        }
+
         $defaultConfig = new \Zend_Config([
-            'format' => $this->getDefaultFormat(),
+            'format' => $format,
```

Comment:
I think this line should stay as it is, because we are constructing the default config here and therefore it should include the default format.